src/components/moj-messages/README.md

```diff
@@ -0,0 +1,85 @@
+# Messages
+
+- [Guidance](https://moj-design-system.herokuapp.com/components/messages)
+- [Preview](https://moj-frontend.herokuapp.com/components/messages)
+
+### Installation
+
+You will need to install the following code at the bottom of `server.js`, just above `module.exports = app;`
+
+```
+// Add filters from MOJ Frontend
+let mojFilters = require('./node_modules/@ministryofjustice/frontend/filters/all')();
+mojFilters = Object.assign(mojFilters);
+Object.keys(mojFilters).forEach(function (filterName) {
+  nunjucksAppEnv.addFilter(filterName, mojFilters[filterName])
+});
+```
+
+## Example
+Below is a typical example of the timeline component in use.
+
+```
+{{ mojMessages({
+  items: [
+    {
+      id: 1,
```

Comment:
Why is an ID needed?

Reply:
It's not. It's optional. But _if_ someone wanted to link to a particular message, however unlikely, they could do so using the ID.